### PR TITLE
Create CI-MOI

### PR DESCRIPTION
**Pull request by AudreyIATI**
Related to [issue 505](https://github.com/org-id/register/issues/505)

### CODE REVIEW

File: lists/CI-MOI [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  "url": "http://http://interieur.gouv.ci/",
+  "description": {
+    "en": "The Ministry of the Interior and Security is the ministerial department of the Ivorian Government responsible for the administration of the territory and public freedoms, the organization of administrative districts, civil status and associations."
+  },
+  "coverage": [
+    "CI"
+  ],
+  "subnationalCoverage": [],
+  "structure": [],
```
Comment (kd-ods) [inline lists/CI-MOI:14]: We'll need something in the 'structure' field about what kinds of organisations are registered with this agency.

Comment (kd-ods) [reply lists/CI-MOI:14]: And that information would also help if added to the 'description' property.

File: lists/CI-MOI [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "licenseDetails": ""
+  },
+  "links": {
+    "opencorporates": "",
+    "wikipedia": "https://fr.wikipedia.org/wiki/Minist%C3%A8re_de_l%27Int%C3%A9rieur_(C%C3%B4te_d%27Ivoire)"
+  },
+  "formerPrefixes": [],
+  "meta": {
+    "source": "Original research",
+    "lastUpdated": "25/3/22"
```
Comment (kd-ods) [inline lists/CI-MOI:44]: Please edit the date to be in 2022-03-25 form for clarity.

File: lists/CI-MOI [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "primary",
+  "access": {
+    "availableOnline": false,
+    "onlineAccessDetails": null,
+    "publicDatabase": "http://interieur.gouv.ci/",
+    "guidanceOnLocatingIds": "The Ministry website is currently undergoing changes, so is it not available for any searches",
+    "exampleIdentifiers": "474/MIS/DGAT/DAG/SDVA",
+    "languages": [FR]
```
Comment (kd-ods) [inline lists/CI-MOI:26]: The FR needs quotation marks around it. So:

["FR"]

**Review by kd-ods — CHANGES_REQUESTED** [review]
There are a few things to correct here. And the directory and filename need to be altered. At the moment it's: lists/CI-MOI. But it needs to be lists/ci/ci-moi.json

Thanks!

File: lists/CI-MOI (hunk 3 above)
Comment (kd-ods) [reply lists/CI-MOI:26]: And it should be lower-case, so ["fr"]

**Comment by AudreyIATI** [comment]
Changes made - let me know if you need anything else from me.

Thanks,
Audrey

**Review by kd-ods — APPROVED** [review]

File: lists/ci/ci-moi.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": {
+    "en": "Ministry of the Interior (Ivory Coast)",
+    "local": "Ministère de l'Intérieur (Côte d'Ivoire)"
+  },
+  "url": "http://http://interieur.gouv.ci/",
```
Comment (andylolz) [inline lists/ci/ci-moi.json:6]: There’s a typo in the URL here – `http://` is duplicated

**Pull request merged**